tests/framework/e2e/etcd_process.go

```diff
@@ -72,6 +77,7 @@ type EtcdServerProcess struct {
 	proxy      proxy.Server
 	failpoints *BinaryFailpoints
 	donec      chan struct{} // closed when Interact() terminates
+	logs       *procLogs
```

Comment:
```suggestion
	logs       procLogs
```
It doesn't need to be a pointer you know?